arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -495,6 +495,12 @@
           },
           "204": {
             "description": ""
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/GenericOkResponse"
+            }
           }
         },
         "x-ms-long-running-operation": true
```

Comment:
As per the swagger rule [M2005](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#naming---swagger-checklist), seems like they are describing that for `delete` operation valid terminal success status codes are "204 or 200 or both for delete".

Either the rule is not complete or something i am misunderstanding. 
@amarzavery / @sarangan12 Could one of you please clarify?